README.md

```diff
@@ -59,14 +59,29 @@ Add in your `config_prod.yml` file, you don't need error notifier when you are i
 ```yml
 # app/config/config_prod.yml
 elao_error_notifier:
-    from: from@example.com
-    to: to@example.com
+    from: from@example.com # required if using "default_mailer" notifier
+                           # ignored if `notifiers.mailer.from` is set
+    to: to@example.com # required if using "default_mailer" notifier
+                       # can be a string or an array or email addresses
+                       # ignored if `notifiers.mailer.to` is set
```

Comment:
I've only left them there for backward compatibility for the configuration at the moment.
I did mention it a little lower down in the readme.

-------

### How to send error via email?

Enter the email addresses you wish to mail from and to (this can either be a string from 1 recipient or an array for multiple) and add the notifier `default_mailer` to your `enabled_notifiers` list.

```yml
# app/config/config_prod.yml
elao_error_notifier:
    # from: from@example.com 
    # to: [ to@example.com ]
    # mailer configurations at root are for backward compatability
    # and may be dropped so it is best to avoid
    enabled_notifiers:
        - default_slack
    notifier:
        mailer:
            from: from@example.com
            to: [ to@example.com ]
    
```